net/openvpn/test.sh

```diff
@@ -4,10 +4,7 @@ case "$1" in
 	"openvpn-mbedtls")
 		openvpn --version | grep "$2.*SSL (mbed TLS)"
 		;;
-	"openvpn-openssl")
-		openvpn --version | grep "$2.*SSL (OpenSSL)"
-		;;
-	"openvpn-wolfssl")
+	"openvpn-openssl"|"openvpn-wolfssl")
 		openvpn --version | grep "$2.*SSL (OpenSSL)"
```

Comment:
wolfssl also uses (OpenSSL) in its version string?

Reply:
> 
> 
> wolfssl also uses (OpenSSL) in its version string?

Yes

openvpn --version
OpenVPN 2.5.3 mipsel-openwrt-linux-gnu [SSL (OpenSSL)] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD]
library versions: wolfSSL 4.7.0
Originally developed by James Yonan
Copyright (C) 2002-2018 OpenVPN Inc <sales@openvpn.net>

grep searches in 1st string

Reply:
OpenVPN 2.5.2 mips-openwrt-linux-gnu [SSL (OpenSSL)] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD]
library versions: wolfSSL 4.7.0
Originally developed by James Yonan
Copyright (C) 2002-2018 OpenVPN Inc <sales@openvpn.net>